scripts/get

```diff
@@ -111,8 +111,10 @@ downloadFile() {
   HELM_DIST="helm-$TAG-$OS-$ARCH.tar.gz"
   DOWNLOAD_URL="https://kubernetes-helm.storage.googleapis.com/$HELM_DIST"
   CHECKSUM_URL="$DOWNLOAD_URL.sha256"
-  HELM_TMP_FILE="/tmp/$HELM_DIST"
-  HELM_SUM_FILE="/tmp/$HELM_DIST.sha256"
+  HELM_TMP_ROOT="/tmp/$(mktemp -d helm-installer-XXX)"
```

Comment:
this will result in the wrong dirpath. `mktemp -d` stands for "make a unique temporary filename". It will create the directory and return an absolute path to the created directory, so it should be

```
HELM_TMP_ROOT="$(mktemp -d helm-installer-XXX)"
```

Reply:
Yes, this is how I initially implemented it, but it seems the documentation is a little bit misleading. Or, which is more likely, we misrepresent _temporary file_ definition. We both assumed that _temporary file_ is the file inside `\tmp` directory, but it's not. The _temporary file_ can be created wherever you want. Let's run the example:
```bash
$ docker run --rm -it ubuntu mktemp -d helm-installer-XXX
helm-installer-rW6
```
As you can see the returned path is relative (**not absolute to /tmp directory**)
So the `/tmp/` prefix is actually required. This is what I figured out during manual tests.

Reply:
Ok, I figured it out. The solution is the missing `-t` switch:
```
-t                  interpret TEMPLATE as a single file name component,
                        relative to a directory: $TMPDIR
```
After applying a `-t` switch the result will be:
```
$ docker run --rm alpine mktemp -dt helm-installer-XXXXXX
/tmp/helm-installer-kIbkhP
```
Additionally, I have extended the directory name template with 3 additional `X`s to make it compatible with Alpine Linux distribution.